changelog/db.changelog-1.13.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+  <changeSet author="VMuthu" id="1">
+    <tagDatabase tag="v1.0.5" />
+  </changeSet>
+  <changeSet author="VMuthu" id="2">
+    <createTable tableName="dim_address">
+      <column name="dimension_key" type="INTEGER" autoIncrement="true" >
+        <constraints nullable="false" primaryKey="true" primaryKeyName="dimension_address_pkey" />
+      </column>
+      <column name="address_id" type="VARCHAR(8)" />
```

Comment:
The columns are different to what is in the ticket? 
Also, address id should be integer as per the ticket?

Reply:
Hi Sid,
The address_id in this table description has beed agreed to keep it as varchar type include the character values also rather than keeping them as integer. This has already been discussed and followed in all the earlier tables. we should take this into any future talbles also.

Reply:
2. we need to have the start date and end date in all the tables to mark the duration of the that value present, Other than that do you find any difference. If not that is what is expected columns.